src/resources/js/views/tooltip.js

```diff
@@ -145,16 +145,23 @@ tribe.events.views.tooltip = {};
 	 * @since 4.9.4
 	 *
 	 * @param {jQuery} $container jQuery object of view container.
+	 * @param {object} data       data object passed from 'afterSetup.tribeEvents' event.
 	 *
 	 * @return {void}
 	 */
-	obj.initTooltips = function( $container ) {
+	obj.initTooltips = function( $container, data ) {
+		var theme = [ 'tribe-events-tooltip-theme', 'tribe-common', 'tribe-events' ];
+
+		if ( -1 !== [ 'photo', 'week', 'map' ].indexOf( data.slug ) ) {
```

Comment:
I would like to see this coming from a Script in Pro.

Reply:
@bordoni fixed this, related to https://github.com/moderntribe/events-pro/pull/1225